draft/mson-parameters-headers.md

```diff
@@ -0,0 +1,216 @@
+---
+RFC: XXXX
+Author: Z
+Status: Draft
+Created: 2015-09-23
+Last Modified: 2015-09-28
+---
+
+# API Blueprint RFC XXXX: MSON Parameters and Headers
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Implementation](#implementation)
+- [Backwards Compatibility](#backwards-compatibility)
+
+## Abstract
+
+This RFC proposes using full [MSON][] syntax for description of URI parameters
+and HTTP headers.
+
+## Motivation
+
+Using MSON syntax for description of URI parameters and HTTP Headers will
+greatly help to unify the API Blueprint syntax and introduce reusability of
+URI parameters and HTTP headers. It will also greatly introduce the reusability
```

Comment:
Yeah, copy/paste is one of my most prevalent errors for sure.

I still haven't played with MSON enough to have more specific feedback, but overall it seems really powerful/helpful to be able to do this. Being able to specify more about the headers/status is really lacking in pure json-schema (and previous to MSON it was harder to do params in blueprint). It seems like this could make both more accessible. Certainly look forward to seeing this land.